fuzzers/frida_gdiplus/src/fuzzer.rs

```diff
@@ -102,10 +102,10 @@ unsafe fn fuzz(options: &FuzzerOptions) -> Result<(), Error> {
 
                 #[cfg(unix)]
                 let mut frida_helper =
-                    FridaInstrumentationHelper::new(&gum, options, tuple_list!(coverage, asan))?;
```

Comment:
Unwrapping the return value of `FridaInstrumentationHelper::new` would keep the same semantics as before, while _allowing_ `Try` here.

Reply:
The thing I'm experiencing is that I can't return Ok(Self) in FridaInstrumentationHelper::new() or the fuzzer crashes. 
I have no idea why it is happening..

Reply:
Hmm, can you check whether the return value is an error after the return? I saw a similar issue in `fuzzer.evaluate_input` in `mutational.rs`. `evaluate_input` would return `Ok((None, None))`, but calling `is_err()` would return True. Is this also happening here?

Reply:
is_err() is False

Reply:
This is so strange, the code should be semantically same.
you can try this with 
Not working one: https://github.com/AFLplusplus/LibAFL/pull/1062/commits/497e5787d5a9f52520fe6428efd16e931e1f5eb5
and Working one:
https://github.com/AFLplusplus/LibAFL/pull/1062/commits/7bd3b23fcda8bd73e61ba2a70737830d8310f755

When it returns Ok(helper) then it stops working 🤔